stable/jenkins/templates/_helpers.tpl

```diff
@@ -275,7 +275,7 @@ Returns kubernetes pod template xml configuration
     <org.csanchez.jenkins.plugins.kubernetes.volumes.{{ $volume.type }}Volume>
   {{- end }}
   {{- range $key, $value := $volume }}{{- if not (eq $key "type") }}
-      <{{ $key }}>{{ $value }}</{{ $key }}>
+      <{{ $key }}>{{ if kindIs "string" $value }}{{ tpl $value $ | quote }}{{ else }}{{ $value }}{{ end -}}</{{ $key }}>
```

Comment:
I don't think the value needs to be quoted in XML. Use of `quote` results in `<mountPath>"/var/myapp/myemptydir"</mountPath>`